ideascube/tests/test_views.py

```diff
@@ -47,6 +49,10 @@ def test_home_page_should_redirect_to_welcome_staff_if_no_staff(app, user):
     assert response.forms['model_form']
 
 
+def test_home_page_should_not_redirect_if_staff_exists(app, staffuser):
+    app.get(reverse('index'), status=200)
```

Comment:
I was surprised that my tests on the home page return a 302.
Then I found that if there is no staffuser the home page redirect and that is tested.
But no test that everything work correctly if there is a staffuser.
So, I add it, but yes, I could create another pr for this.